StarCellBio/settings.py

```diff
@@ -40,7 +40,7 @@
 
 # Default logging configuration
 LOG_LEVEL = 'DEBUG'
-DJANGO_LOG_LEVEL = 'DEBUG'
+DJANGO_LOG_LEVEL = 'WARNING'
```

Comment:
This is safe to leave at debug because it gets overridden during deployment by Ansible.